index.ts

```diff
@@ -1,4 +1,4 @@
-/// <reference path='typings/tsd.d.ts' />
+/// <reference path='typings/definitions.d.ts' />
```

Comment:
Ok, but the side effect would be index.ts is looking for typing in `typings/definition.d.ts`, which only has the DefinitelyTyped stuff, not our overrides?